spec/std/openssl/ssl/socket_spec.cr

```diff
@@ -35,4 +35,23 @@ describe OpenSSL::SSL::Socket do
       client.close
     end
   end
+
+  it "can close connection on servers that don't shut down right" do
+    tcp_server = TCPServer.new(0)
+    server_context, client_context = ssl_context_pair
+
+    OpenSSL::SSL::Server.open(tcp_server, server_context, sync_close: true) do |server|
+      client_done = Channel(Nil).new
+      spawn do
```

Comment:
Could you please put the (entire) server into a separate fiber, since its only auxiliary. The code that's supposed to be tested here is the client connection and that should execute in the main fiber. This helps understanding what this spec does.

Reply:
OK check the latest.